src/queries/queryKeys.ts

```diff
@@ -1,5 +1,6 @@
 export const UserRewardsDetails = 'userRewardsDetails';
 export const addDelegate = 'addDelegate';
+export const addSettings = 'addSettings';
```

Comment:
```suggestion
export const createNotificationSettings = 'createNotificationSettings';
```

Needs to be same as query name.